### PR TITLE
EDX-4113 Convert EdgeX error to XRT error code

### PR DESCRIPTION
**Pull request by weichou1229**
Convert EdgeX error to XRT error code.

### CODE REVIEW

File: xrtmodels/response.go [hunk 1]
```diff
@@ -8,12 +8,13 @@ import (
...
 
 const (
-	// https://github.com/IOTechSystems/xrt/blob/master/include/devsdk/devsdk.h#L27
+	// https://docs.iotechsys.com/edge-xrt21/mqtt-management/mqtt-management.html#general-result-format
 	XrtSdkStatusOk               = 0
 	XrtSdkStatusNotFound         = 1
 	XrtSdkStatusNotSupported     = 2
 	XrtSdkStatusInvalidOperation = 3
 	XrtSdkStatusAlreadyExists    = 7
+	XrtSdkStatusServerError      = 20 // custom error code because XRT haven't defined the Server Error
```
Comment (cloudxxx8) [inline xrtmodels/response.go:17]: use 500 to prevent the future conflict of xrt error code

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]

File: common/xpertconstants.go [hunk 2]
```diff
@@ -66,3 +66,10 @@ const (
 	EtherNetIPMinorRevision     = "MinorRevision"
 	EtherNetIPAddress           = "Address"
 )
+
+const (
+	XpertNodeNameEnvVar      = "XPERT_NODE_NAME"
+	XpertNodeNameFieldKey    = "{" + XpertNodeNameEnvVar + "}"
```
Comment (cloudxxx8) [inline common/xpertconstants.go:72]: use ${XXX} to be comply with Xrt config

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]

File: xrtmodels/response.go (hunk 1 above)
Comment (weichou1229) [reply xrtmodels/response.go]: Updated.

File: common/xpertconstants.go (hunk 2 above)
Comment (weichou1229) [reply common/xpertconstants.go]: Updated.

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**